src/constants/fieldMaps.ts

```diff
@@ -172,6 +174,51 @@ export const awsRegions = [
   },
 ];
 
+export const dateFormats = [
```

Comment:
What about something like this?

```js
export const dateFormats = [...listOfDateFormatStrings].map((format) => ({
  value: format,
  str: `${format} - ${getDateCopy("08/31/2022", {
    dateFormat: format,
  })}`,
}));
```

Lots of copy/pasted strings that we could eliminate!

Reply:
Great idea!